common/types/layer.go

```diff
@@ -76,7 +76,7 @@ func InitGenesisData() {
 		},
 		blockID: GenesisBlockID,
 	}
-	genesisLayer = NewExistingLayer(GetEffectiveGenesis(), []*Ballot{ballot}, []*Block{block})
+	genesisLayer = NewExistingLayer(GetEffectiveGenesis(), Hash32{}, []*Ballot{ballot}, []*Block{block})
```

Comment:
shouldn't this be hash of the actual genesis block?

Reply:
i think there should be no genesis block or ballot, so i am not sure what the point to hash genesis block... this is a tech debt that needs to be removed.
i know that ballot is used in tortoise to bootstrap verifying mode. but i am not even sure why block was added

Reply:
for the sake of consistency. agree that it's otherwise useless.
i am all for removing genesis block and ballot. i wasn't able to do that because i wasn't familiar enough with tortoise test setup.